### ISSUE TITLE
Extraction from JS ES6 files does not work and no debug messages

### ISSUE THREAD

**Issue by jehy** [rubenv/angular-gettext-tools#140]
Please disregard this message and look at the following one.

**Comment by jehy** [comment]
Okay, I found the culprit. Anuglar-gettext-tools don't work with javascript ES6.
We had a piece of code:
```javascript
      async function loadTypes() {
          if ($rootScope.dashboard.types) return;

          const { categories } = await Api.offerTypes();
          $rootScope.dashboard.types = categories;
      }
```
And it siliently died on async function.

I request compatibility with ES6 and printing some debug info - to know that files could not be translated due to some error.

Right now we are going to use babel to downgrade our javascript to processable version.

**Comment by jehy** [comment]
I think that those two commits will be helpful:

https://github.com/bartbutler/angular-gettext-tools/commit/2420853625691f9121e67f508e38ddb8469d66cb
https://github.com/bartbutler/angular-gettext-tools/commit/de9281dcd5efca317aeb72ce3a95dd524304b308

**Comment by gabegorelick** [comment]
There are 2 issues here:
1. We need to provide feedback when parsing fails, and not just silently swallow parser errors.
2. We need to upgrade the espree options to support async/await (which is actually ES7, not ES6/ES2015).

**Comment by gabegorelick** [comment]
It's failing on the async/await, which is not part of ES6/ES2015, but is slated for ES7. espree doesn't support it [1] so there's probably nothing we can do. When async/await is standardized, it will be added to espree, and we should support it without any changes. Until then, use a transpiler like babel before calling angular-gettext.

[1] https://github.com/eslint/espree/issues/140

**Comment by gabegorelick** [comment]
Also, the lack of error message will be fixed when #143 lands. Thanks for reporting this.
